addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -367,6 +368,25 @@ def _process_step_forward_operator(self, discuss_channel):
 
         return posted_message
 
+    def _to_store(self, store: Store, /, *, fields=None):
+        if fields is None:
+            fields = {
+                "answers": None,
+                "message": [],
+                "type": True,
+            }
+        for step in self:
+            data = self._read_format(
+                [f for f in fields if f not in {"answers", "message", "type"}], load=False
+            )[0]
+            if "answers" in fields:
+                data["answers"] = Store.many(step.answer_ids, fields=fields["answers"])
+            if "message" in fields:
+                data["message"] = plaintext2html(step.message) if not is_html_empty(step.message) else False
```

Comment:
This is confusing, it's converting plain text to html, on the condition that the value is not empty html?
It looks like either it should check for empty string rather than empty html (if the original value is not html), or not do a convert (if the original value is already html)?